geode-memcached/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
   testImplementation('org.mockito:mockito-core')
   testImplementation(project(':geode-junit'))
 
+  integrationTestImplementation(project(':geode-membership'))
```

Comment:
What symbol does `geode-membership` bring that `geode-memcached` is consuming?

Reply:
AvailablePort

Reply:
Wow, shame on me. I didn't catch that there was a file-move by looking at the change summary. #approved